### PR TITLE
Improving issue and PR creation by having multiple templates

### PR DESCRIPTION
**Pull request by carstenhag**
**The problem**  
There were some typos in the issue template, also, it was annoying to specify Android/iOS all the time.
Preview:

Screenshot 1

Screenshot 2

**Prerequisites**
I have completed the CLA: YES

Note: In order for us to accept pull requests, please complete the Contributor License Agreement by following instructions in [`CONTRIBUTING`](https://github.com/WhatsApp/stickers/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/android.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Device Manufacturer: [FILL IN]
+
+**Stickers**
+- [ ] My stickers do comply with the sticker requirements (512x512 and < 100 KB)
+- [ ] My tray icon does comply with the requirements (96x96 and < 50 KB)
+- [ ] I am using [WebP] images for stickers.
+- [ ] I am using [PNG] images for stickers.
+
+**Problem Description**  
+<!-- FILL IN WITH DETAILED DESCRIPTION OF THE PROBLEM -->
```
Comment (simonzhexu) [inline .github/ISSUE_TEMPLATE/android.md]: I would prefer this not being in a comment, perhaps something like [ FILL IN WITH DETAILED DESCRIPTION OF THE PROBLEM ]

This way, people will not mistakenly put their description in the comment. 

Same for Reproduction steps, and links

Also for other types.

File: .github/ISSUE_TEMPLATE/android.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+-->
+
+**Specifications**  
+OS Version: [FILL IN]
+Device Manufacturer: [FILL IN]
+
+**Stickers**
+- [ ] My stickers do comply with the sticker requirements (512x512 and < 100 KB)
+- [ ] My tray icon does comply with the requirements (96x96 and < 50 KB)
+- [ ] I am using [WebP] images for stickers.
```
Comment (MarvinJWendt) [inline .github/ISSUE_TEMPLATE/android.md]:
```suggestion
- [ ] I am using **WebP** images for stickers.
```

File: .github/ISSUE_TEMPLATE/android.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] I am using [PNG] images for stickers.
```
Comment (MarvinJWendt) [inline .github/ISSUE_TEMPLATE/android.md]:
```suggestion
- [ ] I am using **PNG** images for stickers.
```

File: .github/ISSUE_TEMPLATE/general.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+-->
+
+**Specifications**  
+OS Version: [FILL IN]
+Device Manufacturer: [FILL IN]
+
+**Stickers**
+- [ ] My stickers do comply with the sticker requirements (512x512 and < 100 KB)
+- [ ] My tray icon does comply with the requirements (96x96 and < 50 KB)
+- [ ] I am using [WebP] images for stickers.
```
Comment (MarvinJWendt) [inline .github/ISSUE_TEMPLATE/general.md]:
```suggestion
- [ ] I am using **WebP** images for stickers.
```

File: .github/ISSUE_TEMPLATE/general.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+- [ ] I am using [PNG] images for stickers.
```
Comment (MarvinJWendt) [inline .github/ISSUE_TEMPLATE/general.md]:
```suggestion
- [ ] I am using **PNG** images for stickers.
```

File: .github/ISSUE_TEMPLATE/ios.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+-->
+
+**Specifications**  
+OS Version: [FILL IN]
+Device Manufacturer: [FILL IN]
+
+**Stickers**
+- [ ] My stickers do comply with the sticker requirements (512x512 and < 100 KB)
+- [ ] My tray icon does comply with the requirements (96x96 and < 50 KB)
+- [ ] I am using [WebP] images for stickers.
```
Comment (MarvinJWendt) [inline .github/ISSUE_TEMPLATE/ios.md]:
```suggestion
- [ ] I am using **WebP** images for stickers.
```

File: .github/ISSUE_TEMPLATE/ios.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] I am using [PNG] images for stickers.
```
Comment (MarvinJWendt) [inline .github/ISSUE_TEMPLATE/ios.md]:
```suggestion
- [ ] I am using **PNG** images for stickers.
```

**Comment by carstenhag** [comment]
Thanks @MarvinJWendt, great comments. I have applied your suggestions.

Hey @simonzhexu, it should be good now!

**Comment by simonzhexu** [comment]
Thanks @carstenhag, Thanks @MarvinJWendt

**Pull request merged**